pkg/config/default.go

```diff
@@ -142,7 +142,7 @@ func DefaultConfig(ctx context.Context, opts ...OpOption) (*Config, error) {
 	cfg.Components[network_latency_id.Name] = nil
 
 	if runtime.GOOS == "linux" {
```

Comment:
We don't need to check if current os is linux.. let the component decide whether to enable..

Reply:
done